components/PluginsList.vue

```diff
@@ -24,11 +24,13 @@ export default {
           >{{ pluginType.name }}</a
         >
       </h2>
+      <!-- eslint-disable vue/no-v-html -->
       <p
         v-if="pluginType.description"
         class="my-8"
         v-html="pluginType.description"
```

Comment:
The assumption is that this is intentional.

Reply:
It's not clear to me if this is intentional. Let me check!

Reply:
It is!